docs/en/observability/apm/view-and-analyze/filter-and-search/cross-cluster-search.asciidoc

```diff
@@ -44,7 +44,20 @@ You can also specify certain clusters to display data from, for example,
 
 There are two ways to edit the default {data-source}:
 
-* In the APM UI -- Navigate to *APM* > *Settings* > *Indices*, and change all `xpack.apm.indices.*` values to
+* In the APM UI -- Navigate to *APM* → *Settings* → *Indices*, and change all `xpack.apm.indices.*` values to
 include remote clusters.
 * In `kibana.yml` -- Update the {kibana-ref}/apm-settings-kb.html[`xpack.apm.indices.*`] configuration values to
 include remote clusters.
+
+[TIP]
+.Exclude data tiers from search
+====
+In a cross-cluster search (CCS) environment, it's possible for different clusters to serve different data tiers in responses.
+If one of the requested clusters responds slowly, it can cause a timeout at the proxy after 320 and 502 responses presented
+as failure toast messages in the UI with no data loaded.
```

Comment:
Is this 320 an error code or a number of seconds? @crespocarlos / @smith ?

I think it should instead be something like this:

> If one of the requested clusters responds slowly, it can cause a timeout at the proxy after 320 seconds. This
> results in 502 error responses, presented as failure toast messages in the UI and no data loaded.

Reply:
Good call out. 320 is the number of seconds.

I suggest being more specific and mentioning `HTTP status code 502`

Reply:
Perfect. Thank you!